lib/ash_rbac/policies.ex

```diff
@@ -99,6 +99,8 @@ defmodule AshRbac.Policies do
     |> Transformer.add_entity([:policies], policy, type: :prepend)
   end
 
+  defp add_action_policies(dsl_state, action_settings) when action_settings == %{}, do: dsl_state
```

Comment:
Just curious, do you prefer when clauses over just having the `%{}` as the parameter?

`defp add_action_policies(dsl_state, %{}) , do: dsl_state`

I think having the parameter name in there is helpful. I also saw this pattern in the past.

`defp add_action_policies(dsl_state, %{} = _action_settings) , do: dsl_state`

Reply:
`%{}` would match on every map. I've done what you suggested above at first, and was confused why tests broke xD

```elixir
iex(1)> defmodule SomeModule do
...(1)> def do_thing(%{}), do: IO.inspect("first_clasue")
...(1)> def do_thing(settings), do: IO.inspect("second_clause")
...(1)> end
warning: variable "settings" is unused (if the variable is not meant to be used, prefix it with an underscore)
  iex:3: SomeModule.do_thing/1

{:module, SomeModule,
 <<70, 79, 82, 49, 0, 0, 6, 8, 66, 69, 65, 77, 65, 116, 85, 56, 0, 0, 0, 195, 0,
   0, 0, 19, 17, 69, 108, 105, 120, 105, 114, 46, 83, 111, 109, 101, 77, 111,
   100, 117, 108, 101, 8, 95, 95, 105, 110, ...>>, {:do_thing, 1}}
iex(2)> SomeModule.do_thing(%{})
"first_clasue"
"first_clasue"
iex(3)> SomeModule.do_thing(%{a: 1})
"first_clasue"
"first_clasue"
```

Reply:
ah, good catch. Inline one is using `=` for matching and yours is using `==` to compare :+1: